model/validation.go

```diff
@@ -669,3 +661,89 @@ func ValidateDestinationPolicy(msg proto.Message) error {
 
 	return errs
 }
+
+func validateProxyAddress(hostAddr string) error {
```

Comment:
Agreed. Maybe the following? Either way, host validation is a different issue from splitting up the hostAddr into host/port tuple./

```golang
if ip := net.ParseIP(host); ip != nil && ip.To4() == nil {
    return fmt.Errors("ipv6 address: %v, host)
}
```